js/app.js

```diff
@@ -1,37 +1,34 @@
 'use strict';
 
-buildArray1();
-buildArray2();
-buildArray3();
-buildArray4();
-buildArray5();
+
+function randNum(min, max) {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
 
 var firstAndPike = {
   name: 'First and Pike',
   minCust: 23,
   maxCust: 65,
   avgCookieSales: 6.3,
-  getCustPerHour: function() {
-    return randNum(this.minCust, this.maxCust);
+  custAndCookiesPerHour: [],
+  getCustPerHour:  function() {
+    this.custPerHour = randNum(this.minCust, this.maxCust);
   },
   getCookiesPerHour: function() {
-    return Math.floor(this.getCustPerHour() * this.avgCookieSales);
+    firstAndPike.getCustPerHour();
+    this.cookiesPerHour = Math.floor(this.custPerHour * this.avgCookieSales);
+    for (var i = 0; i < 16; i++) {
+      var cookie = firstAndPike.getCookiesPerHour();
+      custAndCookiesPerHour.push(cookie[i]);
```

Comment:
you have a linter error here that may help you. When we refer to another key/pair value inside an object, what do we need to put in front of it(hint: this).